test/integration/coscheduling_test.go

```diff
@@ -212,23 +320,43 @@ func TestCoschedulingPlugin(t *testing.T) {
 			t.Logf("Start-coscheduling-test %v", tt.name)
 			testutils.CleanupPods(cs, t, pods)
 			pods = make([]*v1.Pod, 0)
+			// create pod group
+			for _, podInfo := range tt.pods {
+				if podInfo.minAvailable == "" {
+					continue
+				}
+				min, err := strconv.Atoi(podInfo.minAvailable)
```

Comment:
~~Please help to refactor `podInfo` as `minAvailable` is no longer needed to be a field of a pod; instead, it should be a field of PG.~~

Actually, we don't need `podInfo` anymore - we can have the `*v1.Pod` in-placed when initing the subtest:

```go
pods: []*v1.Pod{
    st.MakePod().Name(...).Req(...).Priority(...).Label(...).Obj(),
    ...
}
```

Reply:
`Req()` and `Container()` can not work together well. I have add another func to do this